examples/scenarioMonteCarloAttRW.py

```diff
@@ -104,18 +127,12 @@
 the data can be accessed and tested in different ways.
 This is explained in the example python code comments.
 
-.. note::
-
-    In these Monte Carlo simulations the retained data is stored as the data array with the time
-    information added as the first column.  This is the same retained data format as used
-    with BSK 1.x.
-
 Illustration of Simulation Results
 ----------------------------------
 
 ::
 
-    saveFigures = False, case = 1, show_plots = True, useDatashader = False
+    saveFigures = False, case = 1, show_plots = True, use_bokeh = False
```

Comment:
use `useBokeh` as the variable name to be compliant with BSK lower camel case notation.